### ISSUE TITLE
feature request: click element

### ISSUE THREAD

**Issue by wesamco** [1995eaton/chromium-vim#541]
at the end of the day, it's a graphical web browser, not a text editor.
we need a way to click an element. so we never have to touch the mouse while browsing.
like the `f` key press functionality, (labels all anchors & buttons) 
the `c` key press should label all elements excluding `div` & `span` elements.

here's how to simulate a click, i tested it, it worked.

```
function eventFire(el, etype){
  if (el.fireEvent) {
    el.fireEvent('on' + etype);
  } else {
    var evObj = document.createEvent('Events');
    evObj.initEvent(etype, true, false);
    el.dispatchEvent(evObj);
  }
}
```
usage: `eventFire(document.getElementById('mytest1'), 'click');`

I will try to add this feature and submit a PR, but first I'll have to read and understand the project to write a LOC & contribute.

**Comment by guest73** [comment]
I agree with this. Need functionality similar to Firefox's VimFix when you press 'v' key which is called Place caret at elemet. I really miss this one.

**Comment by KamiGim** [comment]
I'm also agree please make this feature because I cannot follow the link at some websites.

**Comment by lecler-i** [comment]
👍

**Comment by stumash** [comment]
:+1:

**Comment by stumash** [comment]
me too :+1:
